src/seedu/addressbook/data/AddressBook.java

```diff
@@ -78,8 +78,18 @@ private void syncTagsWithMasterList(Person person) {
      */
     public void addPerson(Person toAdd) throws DuplicatePersonException {
         syncTagsWithMasterList(toAdd);
+        // To test if getPrintableString works
```

Comment:
please remove unnecessary code, if you need something later, you can always retrieve from previous commits. RCS \#FTW